monitoring/boot_config_linux.go

```diff
@@ -0,0 +1,186 @@
+/*
+Copyright 2016 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package monitoring
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"os"
+	"regexp"
+	"syscall"
+
+	"github.com/gravitational/satellite/agent/health"
+	pb "github.com/gravitational/satellite/agent/proto/agentpb"
+	"github.com/gravitational/trace"
+)
+
+func int8string(bs []int8) string {
+	b := make([]byte, 0, len(bs))
+	for _, v := range bs {
+		if v == 0 {
+			break
+		}
+		b = append(b, byte(v))
+	}
+	return string(b)
+}
+
+var (
+	reBootConfigParam   = regexp.MustCompile(`(\S+)\=([ym])`)
```

Comment:
I know this is copy/paste code, but is the regex supposed to match 'y' and 'm' or 'y' and 'n' (yes/no)?

Reply:
This refers to a y/n/m, where `m` stands for `module`